server/@types/make-recall-decision-api/models/PpudDetailsResponse.ts

```diff
@@ -36,6 +36,7 @@ export type PpudDetailsSentence = {
 
 export type PpudDetailsOffence = {
   indexOffence?: string,
+  offenceComment?: string,
```

Comment:
Same as with the API PR. We also have indexOffenceComment e.g. 

```
export type PpudUpdateOffenceRequest = {
  indexOffence: string,
  indexOffenceComment?: string,
  dateOfIndexOffence: string,
};
```

Should we make them all either offenceComment/indexOffenceComment for clarity.